src/User32/User32+SafeCursorHandle.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SafeCursorHandle"/> nested type.
+    /// </content>
+    public static partial class User32
+    {
+        /// <summary>
+        /// Represents an HCURSOR handle that can be closed with <see cref="DestroyCursor(IntPtr)"/>.
+        /// </summary>
+        public class SafeCursorHandle : SafeHandle
+        {
+            /// <summary>
+            /// A handle that may be used in place of <see cref="IntPtr.Zero"/>.
+            /// </summary>
+            public static readonly SafeCursorHandle Null = new SafeCursorHandle(IntPtr.Zero, false);
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="SafeCursorHandle"/> class.
+            /// </summary>
+            public SafeCursorHandle()
```

Comment:
Can this be private?

Reply:
I don't *think* so. We have several safe handles defined in the repo so far and we've learned a lot about what works and what doesn't, so mostly I just copy an existing `SafeHandle` and change it to suit the purpose. I forget exactly what is important and what isn't. But I suspect the CLR's p/invoke implementation requires that the default constructor exist (and be public) so it can instantiate this type itself when returning a safe handle.